### ISSUE TITLE
[Enhance] Spark Accordion will use <button> as their triggering elements

### ISSUE THREAD

**Issue by RCopeland** [sparkdesignsystem/spark-design-system#2946]
**User Story:**
As Spark we want to update Spark Accordions from a link to a button so that it is accessible.

**Notes:**
Currently spark's accordions use <a> when, since they don't navigate, should be <buttons>. See rm-12 in the a11y audit.

**AC:**
Vanilla Accordion
1. will see `<a>` changed to `<button>`.
2. Unit Test will pass
